Notebooks/SurveyDesign/get_visit_lists.ipynb

```diff
@@ -151,9 +135,7 @@
   {
    "cell_type": "code",
    "execution_count": 9,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "def write_visitlists_uddf(csvfname, opsimout, NumVisits=NumVistitsuDDF*19 + 170, rootfname='../../data/Run1.1/protoDC2_visits_uDDF',\n",
```

Comment:
`rootfname` will need to be changed now that the relevant csv files are sitting in a separate [repo](https://github.com/LSSTDESC/DC2_visitList).